### ISSUE TITLE
バグ報告: MobileNetV2 の転移学習モデルの最終層でactivation='softmax'が指定されていません。

### ISSUE THREAD

**Issue by YoshihisaNitta** [karaage0703/karaage-ai-book#26]
p.84 下から3行目
  誤: prediction_layer = tf.keras.layers.Dense(NUM_CLASSES)
  正: prediction_layer = tf.keras.layers.Dense(NUM_CLASSES, activation='softmax')

3ーclass classficationなので、最後のDense層の activation に対して 'softmax' を指定しないとうまく動作しないと思います。 ちなみに tf.keras.layers.Dense() の activation の default 値は None です。

学習結果(p.86 のグラフ)が悪いのはこのバグのせいだと思います。

**Comment by karaage0703** [comment]
@YoshihisaNitta さん
やけに性能悪いなとは思っていたのですが…気づけませんでした。
確認の上、サポートサイトで補足させていただきます。貴重なご指摘まことにありがとうございます。

**Comment by karaage0703** [comment]
本件、コードの修正により性能が正しく発揮されること確認できました。
正誤表への追記、コードの修正し、サポートページにも追記を入れましたのでcloseします。
